src/source/feeds.rst

```diff
@@ -31,6 +78,635 @@ XML file.
+
+======= =========================================================================================================
+
+.. index:: phoneNumber
+.. _feed_phoneNumber:
+
+phoneNumber
+"""""""""""
+
+The phone number of the advertiser as international phone number format, e.g.
++31207894561 or as a local phone number, e.g. 06789456612. 0900 and 0800
+numbers are not allowed.
+
+======= =============================
+Example <ad:phoneNumber>+31207894561</ad:phoneNumber>
+======= =============================
+
+.. index:: emailAdvertiser
+.. _feed_emailAdvertiser:
+
+emailAdvertiser
+"""""""""""""""
+
+Flag for allowing emails to be sent to the advertiser.
+
+======= =============================
+Example <ad:emailAdvertiser>true</ad:emailAdvertiser>
+======= =============================
+
+.. index:: regionId
+.. _feed_regionId:
+
+regionId
+""""""""
+
+The region in which the ad is placed. (only applicable for Kijiji Canada)
+A long value from the region tree. Must be the id of a leaf region.
+
+Each ad belongs to one and only one region and region of an ad cannot be updated.
+This field can only set once during creation of an ad.
+
+This field is mandatory if the `region` field of category configuration is ``MANDATORY``
+and optional if the `region` field is ``OPTIONAL``.
+This field must be omitted if the `region` field of category configuration is ``DISABLED``.
+
+Please refer to :ref:`categories` and :ref:`regions`
+
+======= =============================
+Example <ad:regionId>1700274</ad:regionId>
+======= =============================
+
+.. index:: engines
+.. _feed_engines:
+
+engines
+"""""""
+
+Collection of business features used internally for distinguishing between business propositions for the product. 
+If not provided, the default is *classic*.
+Must be one of the models enabled for the seller. Currently multiple values are not supported.
+
+Available options:
+
+========== =============
+Name       Description
+========== =============
+classic    default proposition
+auctions   deprecated
+promotions enables promotions feature
+amnext     ancestor of the classic (still experimental)
+========== =============
+
+======= =========================================================================================================
+Example .. code-block:: html 
+
+            <ad:engines>
+                <engine>promotions</engine>
+            </ad:engines>
+
+======= =========================================================================================================
+
+.. index:: microTip
+.. _feed_microTip:
+
+microTip
+""""""""
+
+A short freeform text with a maximum length of 18 characters, excluding any characters in ``.,/@#<>``.
+It is a feature as part of a package that sellers can purchase (currently available only for Marktplaats tenant).
+It provides extra attention on the ad in the search results.
+
+======= =============================
+Example <ad:microTip>TODAY 15% DISCOUNT</ad:microTipd>
```

Comment:
can you use a different example? We also have a discount feature, and I think that's even supported in feeds?

Reply:
Would SALE be ok here? This 18 chars boundary limits the field adoption to a crosswords lovers club members. You come up with something, and then start counting on fingers if that matches...

Reply:
sure, or smth like 'New Collection' or 'Just In' or 'Last ones!'

Reply:
and yes I did count on my fingers